### ISSUE TITLE
Issue with page `src/pages/start/7.nix-build-flake.mdx`

### ISSUE THREAD

**Issue by kj** [DeterminateSystems/zero-to-nix#175]
As I understand it the `open` command is macOS-specific. The Linux equivalent would be `xdg-open`.

**Comment by lucperkins** [comment]
@kj Good catch! I've opted to just say "open in your browser" and let people decide how to do that. If this proves tricky for people I'm happy to revisit.
